### PR TITLE
Bugfix/export card

### PR DESCRIPTION
**Pull request by TobiasAsk**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       - run: npm run build
       - name: Copy README
         run: cp README.md ./dist/wonka-ui/
-      - run: npm publish --access public
+      - run: npm publish --access public --tag beta
```
Comment (tankefugl) [inline .github/workflows/publish.yml:25]: Do we want to tag it as beta on npm?

File: projects/wonka-ui/src/public-api.ts [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
-export * from './lib/wonka-ui.service';
+export { ChartModule } from './lib/charts/chart.module';
+export { ListModule } from './lib/containers/lists/list.module';
+export { KeyValueUnitListModule } from './lib/containers/key-value-unit-lists/key-value-unit-list.module';
+export { TableModule } from './lib/containers/tables/table.module';
+export { CardModule } from './lib/card/card.module';
+export { FormModule } from './lib/forms/form.module';
+export { InfoPanelModule } from './lib/info/info-panel.module';
+export { NavigationBarModule } from './lib/navigation/navigation-bar/navigation-bar.module';
+// export * from './lib/wonka-ui.module';
```
Comment (tankefugl) [inline projects/wonka-ui/src/public-api.ts:13]: Commented out code.

File: README.md [hunk 3]
```diff
@@ -616,3 +617,22 @@ License][cc-by].
 [cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
 [cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
 =======
+
+## Local development
```
Comment (tankefugl) [inline README.md:621]: Good!

File: .github/workflows/publish.yml (hunk 1 above)
Comment (TobiasAsk) [reply .github/workflows/publish.yml:25]: No!

File: projects/wonka-ui/package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sintef/wonka-ui",
-  "version": "1.1.0",
+  "version": "1.1.1-beta.0",
```
Comment (emmaLitzler) [inline projects/wonka-ui/package-lock.json:3]: This file is auto-generated but it would be cleaner to push the one that is coherent with package.json

Comment (TobiasAsk) [reply projects/wonka-ui/package-lock.json:3]: Indeed, I'm fixing it now!

Good point, will update it.

Comment (emmaLitzler) [reply projects/wonka-ui/package-lock.json:3]: Still not changed.

Comment (TobiasAsk) [reply projects/wonka-ui/package-lock.json]: Fixed now!

**Pull request merged**